gui/backend.py

```diff
@@ -33,7 +34,8 @@ class GameConquerorBackend():
         'sm_get_num_matches' : (ctypes.c_ulong, ),
         'sm_get_version' : (ctypes.c_char_p, ),
         'sm_get_scan_progress' : (ctypes.c_double, ),
-        'sm_set_stop_flag' : (ctypes.c_bool, )
+        'sm_set_stop_flag' : (ctypes.c_bool, ),
+        'sm_process_is_dead' : (ctypes.c_int32, )
```

Comment:
Nice! Just a last detail, this function should now return a `ctypes.c_bool` (it works anyway, but let's be precise when it costs nothing).

This can then be merged, I didn't have time last week, I hope I can do a "GUI warning" next weekend.

Reply:
I see, first return type, then parameters.

So the change should look like this:
```diff
diff --git a/gui/backend.py b/gui/backend.py
index ab8b09f670f9..874f23025f3e 100644
--- a/gui/backend.py
+++ b/gui/backend.py
@@ -34,8 +34,8 @@ class GameConquerorBackend():
         'sm_get_num_matches' : (ctypes.c_ulong, ),
         'sm_get_version' : (ctypes.c_char_p, ),
         'sm_get_scan_progress' : (ctypes.c_double, ),
-        'sm_set_stop_flag' : (ctypes.c_bool, ),
-        'sm_process_is_dead' : (ctypes.c_int32, )
+        'sm_set_stop_flag' : (None, ctypes.c_bool),
+        'sm_process_is_dead' : (ctypes.c_bool, ctypes.c_int32)
     }
 
     def __init__(self, libpath='libscanmem.so'):
```
Function declarations:
```
void sm_set_stop_flag(bool stop_flag);
bool sm_process_is_dead(pid_t pid);